packages/form-state-manager/src/tests/utils/manager-api.test.js

```diff
@@ -180,4 +180,42 @@ describe('managerApi', () => {
     blur('foo');
     expect(managerApi().active).toBeUndefined();
   });
+
+  describe('initializeOnMount', () => {
+    it.skip('should initialize on form level using initialValues', () => {
```

Comment:
skip :hole:

Reply:
intentional, we are not supporting initialValues yet